math/mathcore/inc/Fit/FitUtil.h

```diff
@@ -36,14 +36,14 @@
 #define USE_PARAMCACHE
 
 #ifdef R__HAS_VECCORE
-namespace vecCore{
+namespace vecCore {
 template <class T>
 vecCore::Mask_v<T> Int2Mask(unsigned i)
 {
    T x;
-   for (unsigned j = 0; j < vecCore::VectorSize<T>(); j++) vecCore::Set<T>(x, i, i + 1);
-   return vecCore::Mask_v<T>(x < i);
-   }
+   for (unsigned j = 0; j < vecCore::VectorSize<T>(); j++) vecCore::Set<T>(x, j, j + 1);
+   return vecCore::Mask_v<T>(x <= i);
+}
```

Comment:
No problem with that. I decided to fill it starting from 1  because in my use case I was working with modulos of the vectorsize, so I was mapping that. But i understand is better this way for the more general case.

I understand the casting is to avoid extra casts from the compiler?